readme.md

```diff
@@ -450,3 +461,36 @@ body, .foo {
   [RULE_END]  
 ]
 ```
+
+#### JavaScript Function Value
```

Comment:
So there is no way it could be statically analyzable and the value defined at build time. The more I understand what you did at SC the more I feel how bad that is. Its a pandora box.